engines/ml/xgboost/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
 compileJava.dependsOn(processResources)
 
 processResources {
-    def jnilibDir = "${project.buildDir}/classes/java/main/lib/linux/aarch64/"
+    def jnilibDir = "${project.buildDir}/resources/main/lib/linux/aarch64/"
     outputs.dir file(jnilibDir)
     doLast {
         def url = "https://publish.djl.ai/xgboost/${xgboost_version}/jnilib/linux/aarch64/libxgboost4j.so"
```

Comment:
Did you verify whether changes to the version specified here will trigger task execution?

Reply:
Fixed, added input property